.github/workflows/main.yml

```diff
@@ -0,0 +1,60 @@
+name: main
+
+on:
+  push:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
```

Comment:
What's the reasoning for triggering on these types specifically?  For example, in the samples-ios repo, we trigger CI tests on the `master` branch, or pull requests targeting `master`.

Reply:
No reasoning in particular. A lot of this is copied over from okta-mobile-kotlin's workflows. I just didn't change it, but I can take a look here again.

Reply:
Looked into this more. The default labels that github workflows apply are [opened, reopened, synchronize]. Labeled and unlabeled were added in case e2e label was added or removed from the PR, so that the workflow runs again. Since we are going to be revisiting skipping e2e tests in a different way, I will remove these types for now as well